DigitalLearningSolutions.Web/ServiceFilter/VerifyAdminUserCanViewCourse.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace DigitalLearningSolutions.Web.ServiceFilter
+{
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Helpers;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Mvc.Filters;
+
+    public class VerifyAdminUserCanViewCourse : IActionFilter

```

Comment:
Although this has bore from the need to split view from edit permissions but we now have a slight deviation on naming conventions.
It can be argued that the "...Access..." filters are different to this, so maybe the point is now that those "...Access..." type filters are not so well named if they are dual purpose (i.e. view and edit). Which is totally out of scope for this PR.
What are your thoughts?

Reply:
I think it's less important for the other ones as we don't have this All Centres behaviour with any other items. I'm not sure Access is the best name, but since they can't view or edit anything outside the centre I don't think it's that bad. If we do decide we want to assess the other usages of the Manage/View in a separate ticket, we could also update the name there.